### ISSUE TITLE
Artificial neural networks in the cancer genomics frontier

### ISSUE THREAD

**Issue by gwaygenomics** [greenelab/deep-review#70]
could not resolve doi - http://tcr.amegroups.com/article/viewFile/2647/pdf

**Comment by gwaygenomics** [comment]
very simplistic review discussing some reasonably outdated data processing guidelines for microarray data. Also lightly discusses artificial neural network basics.

It also discusses a few examples of NNs applied to cancer data. I don't believe any of these examples are current issues but they are mostly supervised learning tasks. Three of them are questionable in terms of the amount of data provided to the MLP and appear to be either very easy classification tasks or severely overfit.

1. [Khan et al. 2001](http://dx.doi.org/10.1038/89044) - classify small, round blue cell tumors (SRBCTs) into four distinct childhood tumor classes (neuroblastoma, medulloblastoma, rhabdomyosarcoma, and non-Hodgkin lymphoma). Training set is 63 samples, validation is 25. Build a two layer MLP and achieve 100% accuracy using 96 genes.

2. [Pal et al. 2007](http://dx.doi.org/10.1186/1471-2105-8-5) - using the same SRBCT dataset and only _7_ genes build an MLP architecture with 1 hidden layer of 150 nodes and fuzzy clustering to achieve 100% accuracy on holdout set.

3. [Chang et al. 2011](http://dx.doi.org/10.1007/s00384-011-1279-4) - use MLP to identify a set of 33 miRNA important for distinguishing colorectal cancer from normal tissue
